server/dist/src/main/resources/concord-server.conf

```diff
@@ -255,6 +255,9 @@ concord-server {
         disabledProcessors = [
             "dir"
         ]
+
+        # default branch for import
+        defaultBranch = "master"
```

Comment:
Perhaps we should put `main` as the default value then?

Reply:
I think, currently within the organization, the plan to migrate/adoption of `main` is still in progress.  Hence the plan is to keep it configurable and later we will change it to main

Reply:
It can be `main` in the repo here, and then customized/overridden in a deployed `server.conf` as necessary.

Reply:
changed to main